### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by danieltichiyama**

### CODE REVIEW

File: js/app.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+        planet.appendChild(planetName);
+        filmPlanets.appendChild(planet);
+      }
+
+      sReq.addEventListener("load", sReqListener);
+      sReq.open("GET", obj.results[i].planets[j]);
+      sReq.send();
+    }
+
+    let randomH3 = document.createElement("h3");
```
Comment (taesup) [inline js/app.js]: Choose a better variable name

**Review by taesup — CHANGES_REQUESTED** [review]

File: js/app.js (hunk 1 above)
Comment (danieltichiyama) [reply js/app.js]: variable name changed, ready for review

File: js/app.js [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+        planet.appendChild(planetName);
+        filmPlanets.appendChild(planet);
+      }
+
+      sReq.addEventListener("load", sReqListener);
+      sReq.open("GET", obj.results[i].planets[j]);
+      sReq.send();
+    }
+
+    let superAwesomeAndYetStillRandomH3 = document.createElement("h3");
```
Comment (taesup) [inline js/app.js]: no. change it.

**Review by taesup — CHANGES_REQUESTED** [review]

File: js/app.js (hunk 2 above)
Comment (danieltichiyama) [reply js/app.js]: okay... fine... done :(

**Review by taesup — APPROVED** [review]

**Pull request merged**